tests/conftest.py

```diff
@@ -111,6 +191,54 @@ def kb_one():
     })[0]
 
 
+@pytest.fixture
+def kb_four():
```

Comment:
How about calling this `kb_building`? Using `four` is confusing since there isn't a two or three.